### ISSUE TITLE
LLVM Support

### ISSUE THREAD

**Issue by umgefahren** [matteolutz/umber-lang#3]
Are you planning to add LLVM support instead of manual, although impressive, code generation?
Related: Do you support architectures other than x86-64?

**Comment by matteolutz** [comment]
Yes, I'm definitely planing on implementing LLVM support and ditch my own, in my opinion kind of badly implemented, code generator.

**Comment by umgefahren** [comment]
I think @Zollerboy1 has a great implementation in @juice-lang

**Comment by matteolutz** [comment]
Just wanted to quickly give an update on this issue: Still no LLVM yet, but available compilation targets are now Linux x86-64, Win64 and Macho64 (Check out the new `hello_world_macho64.ub` example).
Maybe I'll also add RISC-V code generation in the near future...
